core/src/main/scala/org/apache/spark/deploy/history/FsHistoryProvider.scala

```diff
@@ -747,6 +747,10 @@ private[history] class FsHistoryProvider(conf: SparkConf, clock: Clock)
         listing.synchronized {
           listing.delete(classOf[LogInfo], rootPath.toString)
         }
+      case _: FileNotFoundException
+          if reader.rootPath.getName.endsWith(EventLogFileWriter.IN_PROGRESS) =>
+        logWarning(s"In-progress file does not exist: ${reader.rootPath}. The application may be " +
```

Comment:
The PR title `Suppress error log for in-progress event log file not found` is misleading.

The intention looks like switching log level from ERROR to WARN, doesn't it?

Reply:
The change also makes it print one line log instead of the whole stack trace